ifrs17/Utils/Extensions.ipynb

```diff
@@ -36,6 +36,33 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Loading external dependencies"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#r \"nuget:morelinq\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "using static MoreLinq.Extensions.ZipLongestExtension;"
```

Comment:
Added this extension for now... It works, but maybe we want our own. I just use ZipLongestExtension of the whole package, as you can get some conflicts otherwise.